modules/openy_gc_personal_training/README.md

```diff
@@ -13,7 +18,11 @@ This module is consist of the 1on1 Meeting entity, additional logic that works w
 
 You cannot use this module without SSL at your website, since it is a webRTC requirement.
 
-### PersonalTraining entity
+## 1on1 Meeting Series
+
+You can create training of series for your clients if they could be grouped by a data range, recurrency and other rules that could be applied to the Date recurrency field. If you change recurrency patterns, module will automatically update/recreate all singe 1on1 Meeetings in your series.
```

Comment:
This needs a little work. Does this rewrite capture everything you're attempting to say?

> You can create training series for your clients using the Date Recurrency field. If you change recurrence patterns, the module will automatically update all 1on1 meetings in your series.